commit 64d3b7dad49d6bd1d8a8d96cc012db45657881a8 (refs/changes/91/2291/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-07-26T15:24:16-07:00 (2 years, 2 months ago)
    
    TOOLS-1824 sdcadm update-other should not do anything for VOLAPI

diff --git a/tools/setup/setup.sh b/tools/setup/setup.sh
index 8262643..f5fc4bd 100644
--- a/tools/setup/setup.sh
+++ b/tools/setup/setup.sh
@@ -288,13 +288,6 @@ upgrade_core_service_to_latest_branch_image "cloudapi" "tritonnfs"
 # volapi" but the VOLAPI zone may need to be updated to the latest version.
 upgrade_core_service_to_latest_branch_image "volapi" "tritonnfs"
 
-echo "Running other experimental migration processes"
-# "sdcadm experimental update-other" takes care of doing things such as adding a
-# VOLAPI_SERVICE key in the sdc's SAPI application metadata, running VMAPI
-# migrations, etc., which are not done by "sdcadm experimental volapi", so this
-# needs to be run every time to make sure everything is up to date.
-sdcadm experimental update-other
-
 # Needed to add additional programs in the HN's GZ, such as sdc-volapi
 upgrade_gz_tools_to_latest_branch_image tritonnfs
 
